crates/bevy_render/src/draw.rs

```diff
@@ -66,6 +66,12 @@ impl Default for Visible {
     }
 }
 
+/// Viewable is used for frustum culling.
+/// Any Sprite or AtlasTextureSprite will have this removed if they are outside the camera frustum and thus not be rendered.
```

Comment:
This comment needs updating with the recent name change (and inversion of logic).